bin/public_interface.py

```diff
@@ -212,13 +215,19 @@ def main() -> None:
     subparsers = parser.add_subparsers(dest="command")
     extract = subparsers.add_parser("extract", help="Extract public interfaces")
     extract.add_argument("--module", help="The module to extract public interfaces", type=str, default="samtranslator")
+    extract.add_argument(
+        "--skipped-modules",
+        help="The modules separated by ',' that should be skipped",
+        type=str,
+        default="samtranslator.internal",
```

Comment:
You can use `nargs='+'` so you don't need the `split`.

Reply:
nargs="*"